### DISCUSSION TITLE
ability to add shortcut to file/folder to launcher

### DISCUSSION THREAD

**Discussion by Simplenuity** [TeamAmaze/AmazeFileManager#4214] [Q&A] [answered]
Thanks for that great file manager!
I'd like to suggest to add the feature of being able to add a shortcut to a file or folder to the launcher.

Thanks for your consideration

**Comment by VishnuSanal** [comment, marked as answer]
@Simplenuity we can currently do that! click a file or folder => press on overflow menu on the right top => select add shortcut.

**Reply by Simplenuity** [reply]
Thanks @VishnuSanal! Good to know.

Duh, didn't check the overflow menu, only the menu next to file/folder where I'd expect that option.
But I guess there's a reason for it, isn't it? "overflow" seems to indicate the amount of entries per menu is limited?
